interpreter/cling/lib/Utils/AST.cpp

```diff
@@ -1441,14 +1457,26 @@ namespace utils {
     LookupResult R(*S, Name, SourceLocation(), Sema::LookupOrdinaryName,
                    Sema::ForRedeclaration);
     Lookup::Named(S, R, Within);
-    if (R.empty())
-      return 0;
+    return LookupResult2Decl<clang::NamedDecl>(R);
+  }
 
-    R.resolveKind();
+  TagDecl* Lookup::Tag(Sema* S, llvm::StringRef Name,
+                           const DeclContext* Within) {
```

Comment:
Indentation

Reply:
thanks. Can I use clang-format to check it thoroughly? I _think_ it's different from ROOT's one.